### DISCUSSION TITLE
Somebody have it running inside Homeassistant?

### DISCUSSION THREAD

**Discussion by bigbasti** [vogler/free-games-claimer#94] [Q&A] [answered]
Just wondering if somebody have already set this image up to run inside home assistant?

**Comment by vogler** [comment]
I don't get what it has to do with Home Assistant. You want to install it as an addon?
I would use Portainer or something similar to manage containers.
See https://www.youtube.com/watch?v=JWE7zl5VvjM
You can use https://github.com/vogler/free-games-claimer/blob/main/docker-compose.yml as a starting point.

**Reply by bigbasti** [reply]
Hi, yes, i know portainer - i was just asking if somebody have already used it that way and maybe had any issues

**Comment by ikrowni** [comment]
I and many others would really appreciate this being published as a home assistant or HACS addon. I think i looked into portainer before and the home assistant devs said wonky stuff can happen messing around with portainer so i always just stayed away from it. But if you could add it to the addon library that would be tits.

**Reply by vogler** [reply]
See https://github.com/vogler/free-games-claimer/issues/149.

**Comment by vogler** [comment, marked as answer]
Tracked in https://github.com/vogler/free-games-claimer/issues/149.
